pom.xml

```diff
@@ -99,21 +101,4 @@
         <module>generic-jms-ra-jar</module>
         <module>generic-jms-ra-rar</module>
     </modules>
-
-    <repositories>
```

Comment:
don't you need to keep to be able to release new versions?

Reply:
This is the configuration for where to pull artifacts from, distributionManagement is the part on where to deploy artifacts to.